import-service/serverless.ts

```diff
@@ -26,7 +26,20 @@ const serverlessConfiguration: AWS = {
             Effect: "Allow",
             Action: ["s3:*"],
             Resource: "arn:aws:s3:::soccer-shop-be-uploaded",
+          },
+          {
+            Effect: "Allow",
+            Action: ["sqs:*"],
+            Resource:"arn:aws:sqs:eu-north-1:034402733310:catalogItemsQueue"
```

Comment:
Not a big deal at all. It's just a thing to think about. 
It's ok for the training, but in prod it's better not to hardcode this values. I have the same issue :)